### PR TITLE
created a seed file and ran test [MP-29]

### PR DESCRIPTION
**Pull request by BernardSimpson**
feat: pathway seed
feat: user seed
feat: challenge
feat: user challenge

### CODE REVIEW

**Comment by BernardSimpson** [comment]
feat: updated seed file by adding the PathChallenge to each challenge

**Review by CraigNorford — APPROVED** [review]

**Review by BsBou — CHANGES_REQUESTED** [review]
Every pathchallenge created has the same challenge_id, chall1.id, meaning the pathway will have 4 challenges, but they will all be the same.
The challenge_id needs to be different for each pathchallenge

**Comment by adrianHards** [comment]
in addition to @BsBou's comments, I recommend reviewing our [style guide](https://craigdoescode.atlassian.net/wiki/spaces/MP/pages/13893859/Code+Style+Guide#Seeds) and making the appropriate adjustments.

**Comment by CraigNorford** [comment]
Also merge with main main into your branch so the testing runs too

**Comment by BernardSimpson** [comment]
Feat: updated seed file by adding loops to create pathways 
feat: updated seed file by adding loops to create challenges

File: db/seeds.rb [hunk 1]
```diff
@@ -1,21 +1,65 @@
...
+    )
+
+    PathChallenge.create!(
+        challenge_id: Challenge.last.id,
+        pathway_id: Pathway.last.id, 
+        order: order  
+    )
+end
+
+20.times do |i|
```
Comment (CraigDoesCode) [inline db/seeds.rb:34]: this part creates 20 pathways

**Pull request closed without merging**